packages/client/components/ThreadedCommentHeader.tsx

```diff
@@ -31,10 +31,10 @@ interface Props {
   meetingId: string
 }
 
-const getName = (comment) => {
-  const {isActive, createdByUserNullable, isViewerComment} = comment
```

Comment:
In https://github.com/ParabolInc/parabol/pull/6042, you added the `createdByUserNullable` alias to `createdByUser` because the type can be null for a comment, but non-null for a poll or task. I believe the alias stops the two fragments from being merged as one?

However, in the demo, it looks like aliasing isn't working as `createdByUserNullable` is always `null` here and in https://github.com/ParabolInc/parabol/blob/8e2f30a09de44a299abd217b63a38cf6cad5201e/packages/client/components/ThreadedCommentBase.tsx#L74

Interestingly, by reverting the change here, `createdByUserNullable` is no longer null in ThreadedCommentBase. 

The demo, tasks, and meeting discussion threads are working well with this change, but perhaps you have an idea why the demo might not work with aliases?

Reply:
ah, you might not see it fail validation because we don't validate persisted queries.
if you change this line to true, you'll probably see the error pop up again: https://github.com/ParabolInc/parabol/blob/fa767a6f719cec03a966d41cc45d1a077a1857fb/packages/server/graphql/executeGraphQL.ts#L57

To fix it in the demo, you could add `createdByUserNullable` to the response & that should fix it

Reply:
Makes sense!